webapp-django/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -74,25 +93,19 @@ def get_entropy(self):
                     yield x + y + z
 
     def find_missing(self, num_workers, date):
-        s3_context = get_s3_context()
-        bucket_name = s3_context.config.bucket_name
-        boto_conn = s3_context._connect()
-
-        check_crashids_for_date = partial(
-            check_crashids, boto_conn=boto_conn, bucket_name=bucket_name, date=date
-        )
+        check_crashids_for_date = partial(check_crashids, date=date)
 
         missing = []
-        entropy = self.get_entropy()
+        entropy_chunked = chunked(self.get_entropy(), 5)
         if num_workers == 1:
-            for result in map(check_crashids_for_date, entropy):
+            for result in map(check_crashids_for_date, entropy_chunked):
                 missing.extend(result)
         else:
             with concurrent.futures.ProcessPoolExecutor(
                 max_workers=num_workers
             ) as executor:
                 for result in executor.map(
-                    check_crashids_for_date, entropy, timeout=300
+                    check_crashids_for_date, entropy_chunked, timeout=WORKER_TIMEOUT
```

Comment:
In early rounds of making this faster, I was exceeding the timeout, so I stuck it in a constant with a comment.